client/src/components/Projects/ProjectsPage.js

```diff
@@ -337,8 +337,9 @@ const ProjectsPage = ({ account, contentContainerRef }) => {
     setFilterText(text);
   };
 
-  const getTimeInMs = date => {
-    return new Date(date).getTime();
+  const getDateOnly = date => {
```

Comment:
In order to keep with filterProject’s convention of checking for false conditions, I had to adjust the getTimeInMs function to remove timestamps. While the project date entries all have timestamp values, the DateRangePicker component defaults all timestamps to 00:00:00. This creates issues, particularly when comparing “equal” dates: For example, Oct. 3rd at 5pm is “greater than” Oct. 3rd at 12am, thus returning false and omitting all end-date entries greater than 00:00:00. (This isn’t an issue with start-date entries because all selected timestamps are “less than” the project timestamps). For this scenario, the most reliable option I’ve found is to convert the date to a string, which omits the timestamp, then convert that string into a Date object for comparison.